image_bootstrap/distros/debian_based.py

```diff
@@ -17,12 +17,20 @@
 
 
 _ETC_NETWORK_INTERFACES_CONTENT = """\
-# interfaces(5) file used by ifup(8) and ifdown(8)
+# This file describes the network interfaces available on your system
+# and how to activate them. For more information, see interfaces(5).
+
+source /etc/network/interfaces.d/*
+
+# The loopback network interface
 auto lo
 iface lo inet loopback
 
+# The primary network interface
 allow-hotplug eth0
 iface eth0 inet dhcp
+# This is an autoconfigured IPv6 interface
+iface eth0 inet6 auto
```

Comment:
Could you drop these two lines from the commit? They seem not needed for working IPv6 access and I did not see the netinst installer add these (or similar) lines for me. The rest I'm on board with :+1:

Reply:
see below...

Reply:
Note to self: https://github.com/hartwork/image-bootstrap/pull/547#issuecomment-2661091924